app/assets/javascripts/discourse/app/widgets/hooks.js

```diff
@@ -112,6 +120,18 @@ WidgetClickHook.setupDocumentCallback = function() {
     }
   };
 
+  $(document).on("mouseover.discourse-widget", e => {
```

Comment:
Do we need to use jQuery here?

Reply:
No we dont, the whole file is following this structure, I can note to fix the whole thing after the release :+1: